features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -3978,4 +3978,9 @@
 
     <SuperTenantAliasInPublicUrl>{{super_tenant.alias_in_public_url}}</SuperTenantAliasInPublicUrl>
 
+    <!-- Configuration settings for Home Realm Discovery -->
+    <HomeRealmDiscovery>
```

Comment:
Having configs for these kind of things will make our life extremely difficult. If this was a bug just fix it.